.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+
+---
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is.]
+
+**What did you expect to happen:**
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+**Environment:**
+
+- Sonobuoy version: 
+- Kubernetes version: (use `kubectl version`):
+- Kubernetes installer & version:
```

Comment:
we've already asked for version in line above

Reply:
This is the *installer* and its version, e.g. kops or kubeadm or whatever.

Reply:
I will have to clarify that one. I am asking for example if its kops and what version of kops.